solc/CommandLineInterface.cpp

```diff
@@ -185,8 +185,8 @@ void CommandLineInterface::handleBinary(string const& _contract)
 {
 	if (m_args.count(g_argBinary))
 	{
-		if (m_args.count(g_argOutputDir))
-			createFile(_contract + ".bin", m_compiler->object(_contract).toHex());
+		if (m_args.count("output-dir"))
```

Comment:
No.  It looks like something left over from merging up that shouldn't be that way.  Will fix.